api/api_sentinel.go

```diff
@@ -0,0 +1,12 @@
+package api
+
+import (
+	"context"
+)
+
+type Sentinel interface {
+	// MethodGroup: Sentinel
+
+	// WatchStart start a watch against the chain
+	WatchStart(context.Context) error
```

Comment:
If the method group is `Sentinel`, the method should be prefixed with `Sentinel`